### ISSUE TITLE
CAT working with High Sierra 10.13 on MBP2011 17"

### ISSUE THREAD

**Issue by mlgboy** [dokterdok/Continuity-Activation-Tool#407]
CAT working on High Sierra on MBP 2011 with upgraded Airport Card

Just to let you guys know that I have all features working with the latest CAT beta including handoff, universal clipboard etc, obviously no Watch unlock as a suitable airport card is not yet available in the UK but it is on its way from the US so I look forward to trying that soon.

Please include the following information.

- OS Version:High Sierra 10.13
- Mac model:"MBP8,3 early 2011 17"
- Dongle or replaced card?:Replaced Airport Card
- Used dongle/card: BCM94331PCIEBT4CAX

**Comment by rkamranb** [comment]
i installed the new patch but hand off still on working on mbp2011 13' late 2011 model 
can u please help?

**Comment by rkamranb** [comment]
which option to choose?

--- OS X Continuity Activation Tool 2.6b6 ---
                 by sysfloat                  
           original by dokterdok              
                                              

---            Legacy kext menu           ---

if you don't know what this menu does please visit the wiki at: TODO:addlink
Select an option:
1) 10.10.5 (971MB)   3) 10.12.2 (1956MB)
2) 10.11.1 (1133MB)  4) Exit
#?

**Comment by mlgboy** [comment]
What Bluetooth are you using?

**Comment by rkamranb** [comment]
how shall i find out?

**Comment by rkamranb** [comment]
Software Versions:
  CoreWLAN:	13.0 (1335)
  CoreWLANKit:	13.0 (1335)
  Menu Extra:	13.0 (1335)
  System Information:	12.0 (1200.3)
  IO80211 Family:	12.0 (1200.12.2)
  Diagnostics:	1.0 (805)
  AirPort Utility:	6.3.8 (638.5)
  Interfaces:
en1:
  Card Type:	AirPort Extreme  (0x14E4, 0xF5)
  Firmware Version:	Broadcom BCM43xx 1.0 (7.21.190.13.1a3)
Locale:	ETSI
  Country Code:	X3
  Supported PHY Modes:	802.11 a/b/g/n
  Supported Channels:	1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 36, 40, 44, 48, 52, 56, 60, 64, 100, 104, 108, 112, 116, 120, 124, 128, 132, 136, 140
  Wake On Wireless:	Supported
  AirDrop:	Supported
  AirDrop Channel:	44
  Status:	Connected

**Comment by Quel11** [comment]
Same setup and working for fine for me. (Also early 2011 mbp / but 15” instead of 17” / same replacement card)
After installing macOS 10.13 all stopped. Installed the latest master of CAT instead of the latest beta by accident. Only airdrop worked. Logged out and in FaceTime on mac. Airdrop  / handoff / phone calls started working.
After last week's "macOS Heigh Sierra 10.13 supplemental update" and reboot, personal hotspot started working again.

**Comment by rkamranb** [comment]
but the continuity is not working for me calls are ok but when i open safari on my iphone it does not show on my mac

**Comment by mlgboy** [comment]
Rkamranb: have you replaced the WiFi/Bluetooth AirPort Extreme card ? If you still have the original card then you will not get full functionality.

**Comment by mlgboy** [comment]
The Bluetooth card must support Le4 “low energy”

**Comment by rkamranb** [comment]
yes i did change it thats why it was working on el capitan and macos sierra its just the macos high sierra its not working

**Comment by rkamranb** [comment]
all the os version before high sierra it was working with my new wifi/bluttooth aiport extreme card

**Comment by mlgboy** [comment]
Did you do a clean install or in place update

**Comment by rkamranb** [comment]
updated it

**Comment by mlgboy** [comment]
Might be worth trying clean install, that’s how I did mine, I have not tried the update on my OS X Sierra SSD drive yet only clean install on a new drive just to see how it would function.

**Comment by rkamranb** [comment]
i dont think i will do a clean install as the only extra hard disk i have i make time machine backup with it

**Comment by rkamranb** [comment]
Update: i signed out of icloud id on mac and singed back in and it started working thanks for help

**Comment by fjdr** [comment]
Hello every one. My MBP 17 early 2011 works fine with Continuity until the GPU give me troubles.
I do a reflow and work again as new. But lost continuity. ( I think the PRAM was reset after I removed the battery)

And now any thing done doesn't work. 

Any suggestion?

**Comment by Quel11** [comment]
Have you tried doing a boot with cmd+r / csrutil disable / reboot/ to get it working again?
